.circleci/config.yml

```diff
@@ -0,0 +1,26 @@
+version: 2.1
+jobs:
+  build:
+    environment:
+      CC_TEST_REPORTER_ID: fb9aa7d1a8b4c0ed1b9cd43ad24e0fb4e1b390f09fb300870019bd10603fb160
```

Comment:
@chekwas88  try not to reveal some sensitive information like this `CC_TEST_REPORTER_ID` cuz it may be prone to the security breach. You may see how to put it into Circle CI environment variables so that it'll be loaded during test runtime.